Formula/pypy.rb

```diff
@@ -70,6 +70,7 @@ def install
     # (like /opt) and symlinking in binaries as needed. Specifically,
     # we want to avoid putting PyPy's Python.h somewhere that configure
     # scripts will find it.
+    system "./configure", "--with-tk=#{MacOS.sdk_path}/System/Library/Frameworks/Tk.framework"
```

Comment:
This will not work since there is no configure script. What you needs is something like adding `ENV.append "CFLAGS", "-I#{MacOS.sdk_path}/System/Library/Frameworks/Tk.framework/Versions/8.5/Headers"` towards the top of the formula.